java/src/main/java/com/powsybl/dataframe/network/adders/SwitchDataframeAdder.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.dataframe.network.adders;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.dataframe.SeriesMetadata;
+import com.powsybl.dataframe.update.UpdatingDataframe;
+import com.powsybl.iidm.network.*;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ * @author Etienne Lesot <etienne.lesot at rte-france.com>
+ * @author Sylvain Leclerc <sylvain.leclerc@rte-france.com>
+ */
+public class SwitchDataframeAdder extends AbstractSimpleAdder {
+
+    private static final List<SeriesMetadata> METADATA = List.of(
```

Comment:
In general: it always has been strange to be to give both voltage level id and bus id. It will be very great to have the possibility to give only busId to create equipments in the network, and to have a way to retrieve the voltage level. It is a bit confusing to expose the complexity of IIDM in python.

Reply:
See discussion synthesis about default values in comment below.